openedx/core/djangoapps/credit/urls.py

```diff
@@ -19,4 +19,10 @@
         credit_provider_callback,
         name="provider_callback"
     ),
+
+    url(
+        r"^v1/provider/info/(?P<provider_id>[^/]+)/?$",
```

Comment:
You can eliminate a lot of duplication in this file by grouping the URL patterns similar to what we do in edx/ecommerce: https://github.com/edx/ecommerce/blob/master/ecommerce/extensions/api/v2/urls.py.